exercises/concept/dnd-char/runtests.jl

```diff
@@ -0,0 +1,46 @@
+using Test
+
+include("dnd-character.jl")
+
+@testset "Randomly generated ability is within range" begin
+    for i in 1:1000
+        @test 3 <= ability() <= 18
+    end
+end
+
+@testset "Ability modifiers" begin
+    @test modifier(3)::Int == -4
```

Comment:
Do we need to coerce to `Int` here? `-4.0 == -4`.

Reply:
I wanted to "force" people to use an integer solution because in the context of the exercise, an Int is the natural type of the modifier. Happy to remove it if you disagree with enforcing it, though.

Reply:
I think this is the kind of thing that mentoring should pick up?

But I'm also fine with this so long as we specify in the instructions that `modifier` must return an `Int`, and then maybe just have one test case for it?

```julia
@test typeof(modifier(7)) <: Integer
```

Reply:
> I think this is the kind of thing that mentoring should pick up?

Concept exercise are mostly not meant to be mentored, so I'd rather have this be part of the tests directly.

> But I'm also fine with this so long as we specify in the instructions that modifier must return an Int, and then maybe just have one test case for it?

I like that idea.